src/components/pages/itemDetail/itemDetail.jsx

```diff
@@ -0,0 +1,414 @@
+    align-self: flex-end;
+    color: white;
+    cursor: pointer;
+`
+
+const activeButtonStyle = css`
+    background: var(--blue)
+`
+
+const commentList = css`
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    gap: 24px;
+`
+
+const commentListStyle =css`
+    display: flex;
+    flex-direction: column;
+    margin-top: 24px;
+    padding-bottom: 12px;
+    border-bottom: 1px solid var(--gray-200);
+`
+
+const commentHeader = css`
+    display: flex;
+    justify-content: space-between;
+`
+
+const backToListButtonStyle = css`
+    background: var(--blue);
+    height: 48px;
+    color: white;
+    border: none;
+    border-radius: 40px;
+    padding: 12px 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 24px auto;
+    cursor: pointer;
+    gap: 8px;
+    
+
+    > img {
+        width: 24px;
+        height: 24px;
+    }
+`
+
+const linkStyle = css`
+    text-decoration: none;
+`
+
+const noCommentImgStyles = css`
+    margin: 10px auto 48px;
+    display: flex;
+    justify-content: center;
+    width: 196px;
+`
+
+const ItemDetail = () => {
+
+    const {productId} = useParams();
+    const [item, setItem] = useState(null);
+    const [comments, setComments] = useState([]);
+    const [commentTextarea, setCommentTextarea] = useState("");
+    const isButtonActive = commentTextarea.trim().length > 0;
+
+    const handleChange = (e) => {
+        setCommentTextarea(e.target.value);
+    }
+
+    useEffect(()=> {
+        async function fetchData() {
+            try {
+                const data = await getItemId(productId);
+                setItem(data);
+            }   catch (err) {
+                console.error(err);
+            }
+        }
+
+        fetchData();
+    }, [productId])
+
+
+    useEffect(()=> {
+        async function fetchData() {
+            try {
+                const data = await getProductComments(Number(productId));
+                setComments(data);
+            } catch (error) {
+                console.error("커멘트 불러오기:", error);
+            }
+        }
+
+        if (productId) {
+            fetchData();
+        }
+        
+    }, [productId]);
+
+    if (!item) return <div>로딩 중...</div>;
+
+    return(
+        <div css={pageWrapper}>
+            <div css={itemDetailWrapper}>
+                <div css={imgSection}>
+                    <img src={item.images || defaultImage}
+                        onError={(e)=> {
+                            e.target.onerror = null;
+                            e.target.src = defaultImage;
+                        }}    
+                    />
+                </div>
+                <div css={itemDetail}>
+                    <div css={itemHeader}>
+                        <div css={titleHeader}>
+                            <h3>{item.name}</h3>
+                            <img src={kebabIcon} css={editIconStyle} />
+                        </div>
+                        <h2>{item.price}원</h2>
```

Comment:
💊 제안
금액의 경우 (,)를 통해 단위가 구분되도록 아래처럼 작성해주세요!
```suggestion
                        <h2>{item.price.toLocaleString()}원</h2>
```